unreleased.rst

```diff
@@ -14,4 +14,5 @@ HEAD — ongoing
 - [DAML Standard Library] **BREAKING CHANGE**: Remove the deprecated modules ``DA.Map``, ``DA.Set``, ``DA.Experimental.Map`` and ``DA.Experimental.Set``. Please use ``DA.Next.Map`` and ``DA.Next.Set`` instead.
 - [Sandbox] Fixed an issue when CompletionService returns offsets having inclusive semantics when used for re-subscription. 
   See `#1932 <https://github.com/digital-asset/daml/pull/1932>`__.
-  
\ No newline at end of file
+  
+- [Sandbox] dalf files are not supported as archives anymore. See `#1610 <https://github.com/digital-asset/daml/issues/1610>`__.
```

Comment:
Might just be me but I don't find this super clear what it this means - what the consequences of this are. I'm guessing it's something like you can no longer provide DALF files in the sandbox's startup parameters, you can only use DAR files now? If so, could you update so that's a bit clearer?

Reply:
I rephrased it. What do you think?

Reply:
I'd go for something more like this: 
```suggestion
- [Sandbox] The Sandbox no longer supports loading from DALF files. You can now only use DAR files. See `#1610 <https://github.com/digital-asset/daml/issues/1610>`__.
```

The idea being to give people enough context to understand where this change is, and what they might need to change.

Also, does this count as a breaking change?

Reply:
I assume it is breaking.

Reply:
I added the BREAKING label.